Formula/rav1e.rb

```diff
@@ -20,9 +21,12 @@ class Rav1e < Formula
   end
 
   def install
+    inreplace "src/capi.rs", 'env!("VERGEN_SEMVER_LIGHTWEIGHT")', "\"#{version}\""
+    inreplace "src/lib.rs", 'env!("VERGEN_SEMVER_LIGHTWEIGHT")', "\"#{version}\""
```

Comment:
Not OK for patching this. It's a detail and makes the formula harder to maintain. Let upstream fix it.

Reply:
Okay, dropped this from the PR.